packages/backend-core/src/events/processors/posthog/tests/PosthogProcessor.spec.ts

```diff
@@ -73,6 +74,8 @@ describe("PosthogProcessor", () => {
     describe("rate limiting", () => {
       it("sends daily event once in same day", async () => {
         const processor = new PosthogProcessor("test")
+        const spy = jest.spyOn(processor.posthog, "capture")
```

Comment:
If we are using this spy in every test, should we move outside and use a single spy?